building/README.md

```diff
@@ -298,13 +298,13 @@ There is a list of commands you can use to get them: on both Ubuntu and MacOS ho
   ```bash
   BINUTILS_VERSION=$(ls $(brew --prefix)/Cellar/binutils/)
   BINUTILS_BIN_PATH=$(brew --prefix)/Cellar/binutils/$BINUTILS_VERSION/bin
-  export PATH=$(brew --prefix make)/libexec/gnubin:$(brew --prefix gnu-sed)/libexec/gnubin:$BINUTILS_BIN_PATH:$PATH
+  export PATH=$(brew --prefix make)/libexec/gnubin:$(brew --prefix gnu-sed)/libexec/gnubin:$PATH:$BINUTILS_BIN_PATH
   ```
 
   and keep it updated, for example by placing the export in the startup script:
 
   ```bash
-  echo "export PATH=\$(brew --prefix make)/libexec/gnubin:\$(brew --prefix gnu-sed)/libexec/gnubin:$BINUTILS_BIN_PATH:\$PATH" >> $HOME/.zshrc
+  echo "export PATH=\$(brew --prefix make)/libexec/gnubin:\$(brew --prefix gnu-sed)/libexec/gnubin:\$PATH:\$BINUTILS_BIN_PATH" >> $HOME/.zshrc
```

Comment:
AFAIK the `$` needs not to be escaped in code blocks, so `\$` should be changed to `$` almost everywhere - it renders invalid command at least on GitHub:
![image](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/3939983/5667be68-bafa-4727-8a51-d42a03c049d0)

Reply:
Edit:
I've missed the intent, this escaping is for shell not to resolve the variables, probably more straight-forward would be to use `'` instead of `"` here (and drop the escapes) to avoid confusion